### PR TITLE
feat: keepPathString option to cancel path string replacement inside text content outputs and make sass external

### PR DESCRIPTION
**Pull request by mindon**
make sass external to fix minify issue

### CODE REVIEW

File: src/index.ts
Comment (BjornTheProgrammer) [inline src/index.ts:84]: Can we get a little more detail on what exactly changes when you enable this?

**Review by BjornTheProgrammer — CHANGES_REQUESTED** [review]
Great work overall! Just really need to have a test or two specifically for the new feature `replacePathStrings`. Can you create an example or two in which this breaks if it is disabled or enabled? Otherwise, I'd be really afraid of breaking this functionality in the future accidentally.

**Comment by BjornTheProgrammer** [comment]
Nice catch on the externals, I can't believe I forgot to make all packages external! Really appreciate the pull request. I just really need to see a test or two for the feature `replacePathStrings` before I can consider merging.

Also, I hadn't realized that you were from Shenzhen. 我是亚利桑那州州立大学的学生，我辅修中文。在亚利桑那州州立大学，我已经学中文一年了。我的中文名字是白彬志。

File: src/index.ts
Comment (mindon) [reply src/index.ts]: var names have been improved

File: src/index.ts
Comment (mindon) [reply src/index.ts]: add test cases on keepPathStrings, using **keepPathStrings** instead of replacePathStrings

**Comment by mindon** [comment]
> Nice catch on the externals, I can't believe I forgot to make all packages external! Really appreciate the pull request. I just really need to see a test or two for the feature `replacePathStrings` before I can consider merging.
> 
> Also, I hadn't realized that you were from Shenzhen. 我是亚利桑那州州立大学的学生，我辅修中文。在亚利桑那州州立大学，我已经学中文一年了。我的中文名字是白彬志。

很高兴认识你，有空欢迎来深圳玩 :-)

**Review by BjornTheProgrammer — APPROVED** [review]

**Comment by BjornTheProgrammer** [comment]
I renamed `keepPathStrings` to `keepOriginalPaths`. Do you think that this is sufficiently descriptive? I'll merge if it is, or rename it to something else.

**Comment by mindon** [comment]
> I renamed `keepPathStrings` to `keepOriginalPaths`. Do you think that this is sufficiently descriptive? I'll merge if it is, or rename it to something else.

it seems ok, thanks 👍

**Pull request merged**
